<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>761</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>INFORMACION DEL VEHICULO:</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Código:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnguardar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>GUARDAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnlimpiar">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>260</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>LIMPIAR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCodigo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Marca</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboMarca">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtColor">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Año:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Combustion</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateVehiculo">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modelo:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboModelo">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>140</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cboCombustion">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>90</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DATOS DEL VEHICULO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>751</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>DATOS DE LOS VEHICULOS:</string>
    </property>
    <widget class="QTableWidget" name="tblVehiculo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>721</width>
       <height>192</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Cod.Vehiculo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Año</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellidos</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Color</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Combustion</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modelo</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
